src/main/java/model/Lotto.java

```diff
@@ -6,15 +6,21 @@
 public class Lotto {
     private List<Integer> lotto = new ArrayList<>();
 
-    public Lotto() {
-    }
-
     public Lotto(List<Integer> lotto) {
-        this.lotto = lotto;
+        LottoNumber lottoNumber = new LottoNumber(lotto);
+        this.lotto = lottoNumber.getLottoNumber();
     }
 
     public List<Integer> getLotto() {
         return this.lotto;
     }
 
+    public Boolean contains(int bonus) {
+        if (lotto.contains(bonus)) {
+            return true;
+        }
+
+        return false;
+    }
```

Comment:
```suggestion
    public Boolean contains(int number) {
        return lotto.contains(number)
    }
```

bonus 변수명으로 한정되지 않도록 
범용적인 변수명을 사용하면 좋을 것 같습니다